tools/fpgainfo/board.c

```diff
@@ -690,3 +690,25 @@ fpga_result fpga_boot_info(fpga_token token)
 out:
 	return res;
 }
+
+fpga_result fpga_image_info(fpga_token token)
+{
+	fpga_result res = FPGA_OK;
+	void *dl_handle = NULL;
+
+	fpga_result (*fpga_image_info)(fpga_token token);
+
+	res = load_board_plugin(token, &dl_handle);
+	if (res != FPGA_OK) {
+		OPAE_MSG("Failed to load board plugin\n", dlerror());
```

Comment:
This needs a format specifier (%s) for the dlerror().  I think that dlerror() can also return NULL, so there needs to also be a NULL check.

Reply:
I've updated the format string, and added a NULL check

Reply:
Could you push the changes?  I don't see them.

Reply:
Sorry, the change was still chilling in my working tree :(